cms/djangoapps/contentstore/tests/test_tasks.py

```diff
@@ -277,17 +283,62 @@ def test_check_broken_links_stores_broken_and_locked_urls(
         ### Check that _save_broken_links_file was called with the correct arguments
         mock_save_broken_links_file.assert_called_once_with(mock_user_task_artifact.return_value, mock.ANY)
 
+    @pytest.mark.skip(reason="This test is not yet implemented")
     def test_user_does_not_exist_raises_exception(self):
-        raise NotImplementedError
+        assert True
 
+    @pytest.mark.skip(reason="This test is not yet implemented")
```

Comment:
Instead of skipping, could you just remove the test case? I think we got enough tests for now.